cloudinit/distros/bsd.py

```diff
@@ -77,12 +76,7 @@ def create_group(self, name, members=None):
                             member, name)
 
     def generate_fallback_config(self):
-        nconf = {'config': [], 'version': 1}
-        for mac, name in net.get_interfaces_by_mac().items():
-            nconf['config'].append(
-                {'type': 'physical', 'name': name,
-                 'mac_address': mac, 'subnets': [{'type': 'dhcp'}]})
-        return nconf
+        self.networking.generate_fallback_config()
```

Comment:
This implementation is now the same as the super-class, so we can drop the method here entirely, I think.

Reply:
true, i was just going step by step with removing wrappers and duplicates